test/defaultMetricsTest.js

```diff
@@ -84,6 +84,7 @@ describe('collectDefaultMetrics', () => {
 				expect(metricValue.labels).toMatchObject(labels);
 			});
 		});
+		expect.assertions(45);
```

Comment:
(I'm not sure if it matters, but usually this is the first line of the `it` function.)